src/AutoRest.CSharp/DataPlane/AutoRest/DataPlaneOutputLibrary.cs

```diff
@@ -207,17 +211,70 @@ public DataPlaneRestClient FindRestClient(OperationGroup operationGroup)
             return restClients;
         }
 
-        private static IEnumerable<LowLevelClientMethod> GetProtocolMethods(OperationGroup operationGroup, RestClientBuilder restClientBuilder, BuildContext<DataPlaneOutputLibrary> context)
+        // Get a Dictionary<operationGroupName, List<methodNames>> based on the "protocol-method-list" config
+        private Dictionary<string, List<string>> GetProtocolMethodsDictionary()
```

Comment:
Does it fail somehow if we have a naming conflict?

Reply:
Could you please give me an example for this?

Reply:
Something like

```
protocol-method-list:
  - Create
  - Update
  - Create
```

Reply:
In this case generator will not fail and it will generate `Create` method only once in the default RestClient because in order to make `IEnumerable<LowLevelClientMethod>`, [here](https://github.com/ShivangiReja/autorest.csharp/blob/protocol-method-list/src/AutoRest.CSharp/DataPlane/AutoRest/DataPlaneOutputLibrary.cs#L277) we are iterating all the operations only once and checking that if the operation name exists in the config or not.